docs/build/configure/webhooks.md

```diff
@@ -74,11 +74,19 @@ If you prefer to configure your webhook with raw JSON instead of the config buil
 ```json {class="line-numbers linkable-line-numbers"}
   "webhooks": [
     {
-      "url": "<Insert your own cloud function or lambda URL for sending the event>",
+      "url": "<Insert your own cloud function or lambda URL for sending the event>?data_type=binary",
```

Comment:
will the `data_type` bit be removed before the URL is called? This feels a bit odd

Reply:
@maxhorowitz Sky is currently out, do you mind answering this? Then I might take over merging this PR for her

Reply:
```suggestion
      "url": "<Insert your own cloud function or lambda URL for sending the event>",
```
Not needed. That stuff gets auto populated with the correct data type depending on the file that is uploaded to our cloud.